ydb/core/mind/hive/hive.h

```diff
@@ -220,6 +220,42 @@ inline std::tuple<ResourceTypes...> GetStDev(const TVector<std::tuple<ResourceTy
     return tuple_cast<ResourceTypes...>::cast(st_dev);
 }
 
+template <typename K, typename V>
+std::unordered_map<V, K> MakeReverseMap(const std::unordered_map<K, V>& map) {
+    std::unordered_map<V, K> result;
+    for (const auto& [k, v] : map) {
+        result.emplace(v, k);
+    }
+    return result;
+}
+
+const std::unordered_map<TTabletTypes::EType, TString> TABLET_TYPE_SHORT_NAMES = {{TTabletTypes::SchemeShard, "SS"},
+                                                                                  {TTabletTypes::Hive, "H"},
+                                                                                  {TTabletTypes::DataShard, "DS"},
+                                                                                  {TTabletTypes::ColumnShard, "CS"},
+                                                                                  {TTabletTypes::KeyValue, "KV"},
+                                                                                  {TTabletTypes::PersQueue, "PQ"},
```

Comment:
May be it would be a good idea to move `TABLET_TYPE_SHORT_NAMES` and `TABLET_TYPE_BY_SHORT_NAME` mapping definitions to `hive.cpp` (along with supporting function `MakeReverseMap()`).